### DISCUSSION TITLE
Chaining Jobs More Easily

### DISCUSSION THREAD

**Discussion by 161siegels** [dagster-io/dagster#24317] [Q&A] [answered]
Is there a way to chain jobs (or schedules) so one job (or schedule) runs after the next. I know how to do this for ops (just pass dependency as an argument) but I am curious about how to do this for a job or a schedule.  example 

```
sched_a = ScheduleDefinition( job=job_a, cron_schedule="0 2 * * *" ) 
sched_b = ScheduleDefinition( job=job_b)
```

How do I make b run once a is complete? Also, I know that sensors can achieve this but I may not always want b to run after a. Can I do this without sensors? Thanks!

**Comment by garethbrickman** [comment, marked as answer]
[Run status sensors ](https://docs.dagster.io/concepts/partitions-schedules-sensors/sensors#run-status-sensors) are the recommended feature to use for this. You can use conditional logic to control when a new RunRequest is yielded.
